pipe_gaps/queries/ais_messages.py

```diff
@@ -74,20 +74,27 @@ def __init__(
         source_messages: str = DB_TABLE_MESSAGES,
         source_segments: str = DB_TABLE_SEGMENTS,
         ssvids: list = None,
+        filter_overlapping_and_short: bool = False
     ):
         self._start_date = start_date
         self._end_date = end_date
         self._source_messages = source_messages
         self._source_segments = source_segments
         self._ssvids = ssvids
+        self._filter_overlapping_and_short = filter_overlapping_and_short
 
     def render(self):
+        overlapping_and_short = ""
+        if self._filter_overlapping_and_short:
+            overlapping_and_short = "and not overlapping_and_short"
+
```

Comment:
We don´t have any release yet, so we are not breaking any previous version.
We are in the way of deprecating this filter, so making this configurable is to be able to test it easily.
With the highest probability, **this will be always set to false**, so it will be the default.

Reply:
OIC, it makes sense then 👍